charts/observation-svc/values.yaml

```diff
@@ -173,7 +173,7 @@ fluentd:
   # -- Flag to toggle deployment of Observation Service fluentd
   enabled: false
 
-  nameOverride: obs-fluentd
+  nameOverride: fluentd
```

Comment:
im not sure if there will be a scenario as such, if one helm release has both an observation service and logwriter (possible?) 

both the fluentd deployed by obs svc and logwriter will possibly clash.

Reply:
I think it's possible, but in dataset service we'll prefix each release name accordingly to avoid this issue.